### PR TITLE
Remove and ignore .DS_Store files

### PR DESCRIPTION
**Pull request by selbekk**
## 📝 Description

This pull request removes existing .DS_Store files and add them to .gitignore for ignoring future files.

.DS_Store files are just garbage files from MacOS, and we don't need them in our repo!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@ yarn-error.log
 tsconfig.tsbuildinfo
 .next
 .idea
+.DS_Store
```
Comment (m4x3d) [inline .gitignore:18]: This entry exists already in this file, see line 86.

**Review by m4x3d — COMMENTED** [review]
Hey @selbekk, thanks for cleaning up that unnecessary DS_Store files! :heart: 

The .DS_Store files are already in the gitignore but possible someone does not remove the existing files when adding this to the gitignore.

**Comment by selbekk** [comment]
There we go, should be good now. Thanks for spotting my error @m4x3d :)

**Comment by nikolovlazar** [comment]
Thanks for this PR @selbekk!

**Pull request merged**